index.js

```diff
@@ -400,23 +400,39 @@ async function run() {
     const enableECSManagedTags = enableECSManagedTagsInput.toLowerCase() === 'true';
     const propagateTags = core.getInput('propagate-tags', { required: false }) || 'NONE';
 
-    // Register the task definition
-    core.debug('Registering the task definition');
-    const taskDefPath = path.isAbsolute(taskDefinitionFile) ?
-      taskDefinitionFile :
-      path.join(process.env.GITHUB_WORKSPACE, taskDefinitionFile);
-    const fileContents = fs.readFileSync(taskDefPath, 'utf8');
-    const taskDefContents = maintainValidObjects(removeIgnoredAttributes(cleanNullKeys(yaml.parse(fileContents))));
-    let registerResponse;
-    try {
-      registerResponse = await ecs.registerTaskDefinition(taskDefContents);
-    } catch (error) {
-      core.setFailed("Failed to register task definition in ECS: " + error.message);
-      core.debug("Task definition contents:");
-      core.debug(JSON.stringify(taskDefContents, undefined, 4));
-      throw(error);
+    let taskDefArn = null;
+
+    // Of taskDefContent starts with arn: then we assume it is a task definition ARN
+    if (taskDefinitionContent.startsWith("arn:")) {
```

Comment:
Thank you for this PR!

I am wondering if it would be a good idea to introduce another variable for `taskDefinitionArn` instead of re-using existing variable? This could ease error handling by helping us return proper error messages back to users.

We have something similar implemented in the `amazon-ecs-render-task-definition` repo as well: 
https://github.com/aws-actions/amazon-ecs-render-task-definition/blob/master/dist/index.js#L32